lua/entities/gmod_tardis/modules/sh_health.lua

```diff
@@ -112,7 +112,8 @@ if SERVER then
         self:SetLocked(true,nil,true)
         local maxhealth = TARDIS:GetSetting("health-max")
         local curhealth = self:GetData("health-val")
-        local time = CurTime() + ( math.Clamp((maxhealth - curhealth) * 0.1, 1, 60) )
+        local maxtime = TARDIS:GetSetting("long_repair") and 60 or 10
+        local time = CurTime() + ( math.Clamp((maxhealth - curhealth) * 0.1, 1, maxtime) )
```

Comment:
The 0.1 should also probably be scaled appropriately, maybe 0.015 when long repair is disabled